src/components/Console.jsx

```diff
@@ -17,11 +19,18 @@ export default function Console({
     return null;
   }
 
+  const mostRecentCompiledProject = compiledProjects.last();
+  const compiledProjectKey = get(mostRecentCompiledProject, 'timestamp', 0);
```

Comment:
Let’s push this down into a selector—the `Console` component shouldn’t need to know about the entire collection of compiled projects.